certbot-apache/local-oldest-requirements.txt

```diff
@@ -1,3 +1,3 @@
 # Remember to update setup.py to match the package versions below.
-acme[dev]==0.29.0
-certbot[dev]==1.6.0
\ No newline at end of file
+acme[dev]==1.8.0
```

Comment:
Why are we bumping the acme version? Because of the transitive dependency? Pretty sure we don't usually do it that way.... maybe we should...

Reply:
:shrug: I recall I had problems with only bumping `certbot` initially, but it seems to be okay now, so I've reverted this part.

Reply:
Yea so it fails in CI on `nginx-oldest`:

    2021-04-22T07:58:00.8493293Z pkg_resources.VersionConflict: (acme 0.29.0 (/home/vsts/work/1/s/.tox/nginx-oldest/lib/python3.6/site-packages), Requirement.parse('acme>=1.8.0'))

Reply:
Ok, looks like tox installs all our packages here, and given python dependency management, that's enough of an explanation for me.